ops/framework.py

```diff
@@ -1057,10 +1057,10 @@ class BoundStoredState:
     if TYPE_CHECKING:
         # to help the type checker and IDEs:
         @property
-        def _data(self) -> StoredStateData: ...  # noqa, type: ignore
```

Comment:
This, and the next change, are drive-bys. I weirdly get ruff complaining about these but only some of the time that it's run (I haven't dug into why that is). They do not seem to be necessary, so simplest to just remove.